pkg/objstore/gcs/gcs.go

```diff
@@ -32,6 +38,8 @@ type Bucket struct {
 	opsTotal *prometheus.CounterVec
 }
 
+var _ objstore.Bucket = &Bucket{}
```

Comment:
Can you add a comment on why is this here?

Reply:
I find it a bit confusing that its initialised if its not used or needed for a specific side effect. Especially if all types of buckets are covered by tests.

Reply:
makes sense.